crypto/ct/ct_locl.h

```diff
@@ -151,6 +151,13 @@ __owur int SCT_CTX_set1_issuer_pubkey(SCT_CTX *sctx, X509_PUBKEY *pubkey);
 __owur int SCT_CTX_set1_pubkey(SCT_CTX *sctx, X509_PUBKEY *pubkey);
 
 /*
+ * Verifies an SCT with the given context.
+ * Returns 1 if the SCT verifies successfully; any other value indicates
+ * failure. See EVP_DigestVerifyFinal() for the meaning of those values.
+ */
+__owur int SCT_CTX_verify(const SCT_CTX *sctx, const SCT *sct);
```

Comment:
Strictly speaking we don't usually capitalise internal names (although there are plenty of historic examples where we break that rule). However since the rest of this file also breaks that rule, I guess we let it slide. We can always fix it later as these are purely internal symbols.